dev/format_all.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+"""Erlang formatter for CouchDB
+Warning: this file needs to run from the CouchDB repo root.
+USAGE: ERLFMT_PATH=<path_to_erlfmt> python3 dev/format_all.py
+"""
+
+import os
+import subprocess
+
+from format_lib import get_source_paths
+
+
+def get_hashes():
+    hashes = {}
+    for item in get_source_paths():
+        if item["is_source_path"]:
+            beam_path = f"{item['dirname']}/ebin/{item['filename']}.beam"
+            hashes[item['raw_path']] = subprocess.run(["md5sum", beam_path],
+                                                      encoding="utf-8",
+                                                      capture_output=True).stdout
+        else:
+            # command = ["erl",
```

Comment:
Do we still need commented code? Did you have files matching this branch?

Reply:
The thing is, I'd want to have all `erl` files native to this repo in `erlfmt`'s check, but I couldn't get the checksums to match for files in `test`. Hoping that someone could make it work later, I'd like to leave the commented-out code there.